filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2842,7 +2843,8 @@ void OpenGLDriver::readPixels(Handle<HwRenderTarget> src,
         PixelBufferDescriptor& p = *pUserBuffer;
         auto& gl = mContext;
         gl.bindBuffer(GL_PIXEL_PACK_BUFFER, pbo);
-        void* vaddr = glMapBufferRange(GL_PIXEL_PACK_BUFFER, 0,  p.size, GL_MAP_READ_BIT);
+        void* vaddr = glMapBufferRange(GL_PIXEL_PACK_BUFFER, 0,  p.size,
```

Comment:
According to https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glMapBufferRange.xhtml `glMapBufferRange` generates `GL_INVALID_OPERATION` if both `GL_MAP_READ_BIT` and `GL_MAP_UNSYNCHRONIZED_BIT` are set.

Reply:
Heh. What do you know! Good catch.